Gemfile.lock

```diff
@@ -281,4 +281,4 @@ DEPENDENCIES
   webmock
 
 BUNDLED WITH
-   1.14.5
+   1.15.1
```

Comment:
Asked in Slack:

Carlos:
"We have recently upgraded bundler in all applications to 1.15.1: github.com/alphagov/govuk-puppet/pull/6103/files

so when people run puppet on the local VM we will all be on the same version, so I think in this case it’s fine to leave the update in Gemfile.lock"

So seems good 👍